pom.xml

```diff
@@ -248,13 +249,16 @@
         <mina.version>2.0.22</mina.version>
         <netty.version>4.1.77.Final</netty.version>
         <nimbus-jose-jwt.version>8.14.1</nimbus-jose-jwt.version>
-        <nodejs.version>v12.18.2</nodejs.version>
+        <nodejs.version>v12.20.2</nodejs.version>
         <okhttp.version>2.7.5</okhttp.version>
         <opensaml.version>3.4.5</opensaml.version>
         <pac4j.version>4.5.6</pac4j.version>
         <postgresql.version>42.3.3</postgresql.version>
         <mysql.version>8.0.28</mysql.version>
         <protobuf.version>3.16.1</protobuf.version>
+        <powermock.version>2.0.9</powermock.version>
+        <purejavacomm.version>0.0.11.1</purejavacomm.version>
+        <pty4j.version>0.11.4</pty4j.version>
```

Comment:
Note that this is EPLv1 licensed. This is a weak copy-left licence under what ASLv2 considers Category B. We will need to meet the conditions in https://www.apache.org/legal/resolved.html#category-b

Reply:
Adding pty4j and anything else with this license to the entry in LICENSE file where we are calling out Jetty and Jerico is likely enough.

Reply:
Thanks, I thought I had checked the license. Let me add those entries.

Reply:
So, we already have Eclipse license for Jetty, I just added Pty4j to the list using that license (without duplicating the entire license text)